lib/active_model_serializers/test/schema.rb

```diff
@@ -1,3 +1,5 @@
+require 'minitest/autorun'
```

Comment:
👅 Thanks for reading my comment in a positive light.  Without knowing what problem you were working around, I wasn't sure what to write and didn't word it so well.  autorun, specificaly, is almost certainly not what you want.  no special require should be necessary here, except maybe ensure `Minitest::Assertion`  is defined.

I usually use ✂️ and see it used to mean 'snip'.